client/scss/components/_userbar.scss

```diff
@@ -286,14 +290,20 @@ $positions: (
     max-height: 60vh;
   }
 
+  .w-dialog__header {
+    display: flex;
+    align-items: baseline;
```

Comment:
Is this intended? I assumed the original intention was to center this vertically, e.g.

<img width="966" alt="image" src="https://user-images.githubusercontent.com/6379424/215071755-678112c6-d700-4165-9575-445ea1f4e109.png">

Though to achieve this you'll need to change this to:

```suggestion
    align-items: center;
```

and then add the following:

```css
.w-dialog__title {
    margin-bottom: 0;
```

```css
.w-dialog__subtitle {
    margin-bottom: 0;
}
```

Reply:
yes, intended: center gives not an optimal result: 
![image](https://user-images.githubusercontent.com/51043550/215076968-2a33c43c-a35a-454a-a262-1aed6539a43e.png)
while baseline looks better:
![image](https://user-images.githubusercontent.com/51043550/215077182-93be3a01-4446-4ca2-8601-e12c0491b831.png)

Reply:
Yes, but have you tried removing the bottom margins on the `title` and `subtitle`?

Reply:
yep - just found that there are bottom margins in the dialog component, so I overwrote them for our modal. now works perfectly